src/main/java/com/github/paweladamski/httpclientmock/condition/UrlEncodedFormCondition.java

```diff
@@ -0,0 +1,77 @@
+package com.github.paweladamski.httpclientmock.condition;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.hamcrest.Matcher;
+
+import com.github.paweladamski.httpclientmock.Request;
+
+/**
+ * Tests the request body for URL-encoded parameters.
+ * @author Michael Angstadt
+ */
+public class UrlEncodedFormCondition implements Condition {
```

Comment:
Please add method `debug` and use it in `Rule#debug`, you can see example implementation in `UrlCondition` class.

Reply:
Fixed in 275ea2b73704e3ca0becfca11d924fa69ecf5d06.